tm2/pkg/std/account.go

```diff
@@ -38,6 +38,10 @@ type Account interface {
 	String() string
 }
 
+type AccountRestricter interface {
```

Comment:
I'm trying to understand if we even need this, given that all accounts are `GnoAccount`s. What do you think?

Reply:
We introduced this interface, shared between BaseAccount and GnoAccount, because we need to check the restriction bit flag in SendCoins and InputOutputCoins in the tm2/bank keeper, which should not import or access the concrete type GnoAccount. Otherwise, we would need to introduce a new gnobank SDK module that embeds the bank keeper and handler specifically for gno.land, just to check the restriction bit flag in the GnoAccount struct. The latter option is less favored.

Reply:
IsUnrestricted() bool only. 
!IsUnrestricted() != IsRestricted() see my other comment somewhere

Reply:
AccountUnrestricter